src/domain/Inputs/SingleDateInput/SingleDateInput.tsx

```diff
@@ -76,6 +82,10 @@ class SingleDateInput extends React.PureComponent<ISingleDateInputProps, ISingle
           onFocusChange={this.handleFocusChange}
           isOutsideRange={isDisabledForDate ? isDisabledForDate : this.handleDisabledDateRange}
           renderMonthText={this.renderMonthText}
+          calendarInfoPosition='top'
+          renderCalendarInfo={showMonthPicker ? this.renderMonthPicker : undefined}
+          /** gross hack to change displayed month when date does https://github.com/airbnb/react-dates/issues/1320 */
+          key={value?.toISOString()}
```

Comment:
https://github.com/airbnb/react-dates/issues/1320
`a slight improvement over these workarounds would be setting the date's month as the key so that it flicks only on month change.`

Also it might be better to only pass this if the month picker is being used

Reply:
we don't just want month though, we are also changing the year

Reply:
Yeah I know, which is why you'd need the year as well; looks like you've done it already though so w/e